xml/installation-ardana-manila.xml

```diff
@@ -176,11 +441,11 @@ on &clm;:
 --protocol icmp
 &prompt.ardana;openstack security group rule list manila-security-group (verify manila security group)
 &prompt.ardana;openstack keypair create --public-key ~/.ssh/id_rsa.pub mykey
-&prompt.ardana;neutron net-create n1
-&prompt.ardana;neutron subnet-create --name s1 n1 <replaceable>11.11.11.0/24</replaceable>
-&prompt.ardana;neutron router-create r1
-&prompt.ardana;neutron router-interface-add r1 s1
-&prompt.ardana;neutron router-gateway-set r1 ext-net
+&prompt.ardana;openstack network create n1
+&prompt.ardana;openstack subnet create --name s1 n1 <replaceable>11.11.11.0/24</replaceable>
```

Comment:
`openstack subnet create s1 --network n1 <replaceable>--subnet-range 11.11.11.0/24</replaceable>`

Reply:
updated, does this mean that the Cloud 9 docs need to be updated as well, or was this changed between Pike and Rocky?

Reply:
I don't think it changed. It's more a matter of whether you use the `neutron` CLI or the `openstack` client. If we use the `neutron` client in the Cloud 9 docs we should update those as well. Whenever possible we should be advocating the `openstack` client.